tests/erlang_tests/test_bs_utf.erl

```diff
@@ -49,6 +55,49 @@ test_put_utf8() ->
     ok = assert_badarg(fun() -> <<XF3/utf8>> end),
     ok.
 
+test_get_utf8() ->
```

Comment:
We are obviously missing test cases for the invalid UTF-8 sequences mentioned above.

Reply:
Added now.